packages/manager/modules/pci/src/projects/project/storages/objects/add/add.routing.js

```diff
@@ -1,25 +1,50 @@
+import find from 'lodash/find';
 import filter from 'lodash/filter';
 import get from 'lodash/get';
 import map from 'lodash/map';
+import set from 'lodash/set';
 import some from 'lodash/some';
 
+import { OBJECT_CONTAINER_OFFERS } from '../../containers/containers.constants';
+
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('pci.projects.project.storages.objects.add', {
     url: '/new',
     component: 'pciProjectStorageContainersAdd',
     resolve: {
       regions: /* @ngInject */ (PciProjectRegions, projectId) =>
         PciProjectRegions.getAvailableRegions(projectId).then((regions) => {
-          const supportedRegions = filter(regions, (region) =>
-            some(get(region, 'services', []), {
-              name: 'storage',
+          // Remove code start - temporary fix added until the API returns the correct data
```

Comment:
The code commented as 'to be removed' solves the following 2 issues temporarily:

1. The API gives the name of the region as PREPROD.regionName. This breaks the UI. Hence, we store this name temporarily in the `actualName` variable. This can be removed once the API gives us the proper region name.
2. There is a PR on the API side, that adds the 'storage-high-perf' service to the region. For now, this is being added in the UI side. Can be removed once the API handles it

Reply:
Removed :)